packages/peregrine/lib/talons/CategoryTree/__tests__/useCategoryTree.spec.js

```diff
@@ -140,10 +141,76 @@ test('calls updateCategories when data changes', () => {
     expect(updateCategories).toHaveBeenNthCalledWith(1, data.category);
 });
 
-test('returns a map of child categories', () => {
-    createTestInstance(<Component {...props} categories={categories} />);
+describe('child categories', () => {
+    test('is a Map', () => {
+        createTestInstance(<Component {...props} categories={categories} />);
+
+        expect(log).toHaveBeenNthCalledWith(1, {
+            childCategories: expect.any(Map)
+        });
+    });
```

Comment:
Now that `childCategories` isn't the only thing returned from the talon, this test is failing.

Usually you could just add `data` here like `data: expect.any(Object)`, but since `data` can be `null`, this becomes slightly more complicated. 

We do this a couple of ways currently:

**Snapshot testing**

```js
const talonProps = log.mock.calls[0][0];
expect(talonProps).toMatchSnapshot();
```

**Property existence testing**

```js
const talonProps = log.mock.calls[0][0];
const expectedProperties = ['whatever', 'properties', 'you', 'expect'];
const actualProperties = Object.keys(talonProps);
expect(actualProperties.sort()).toEqual(expectedProperties.sort());
```

Reply:
Also, since this test case is no longer exclusively testing `childCategories`, please move it out of the "child categories" `describe` block above and rename it "returns the correct shape" for consistency with similar tests across the codebase 👍 .